packages/reaction-product-variant/client/templates/products/productDetail/productDetail.js

```diff
@@ -199,17 +199,20 @@ Template.productDetail.events({
       template.$(".title-edit-input").focus();
     }
     const variants = ReactionProduct.getVariants(self._id);
-    for (let variant of variants) {
-      let index = _.indexOf(variants, variant);
+    variants.forEach((variant, index) => {
```

Comment:
Ah, yes, you're right, but still I'm thinking `For..of` is more for objects iteration.
Anyway, if you guys decide to leave `for...of`, I'll try to remember this and not change them in future.